Sources/Util/Util.swift

```diff
@@ -22,6 +22,17 @@ public func isAnImage(_ file: String) -> Bool {
            file.hasSuffix("png")
 }
 
+public func getRange(of rawRange: String, length contentLength: Int) -> Range<Int> {
+  let splitRange = rawRange.components(separatedBy: "-")
+
+  let rangeEnd = Int(splitRange.last!) ?? contentLength - 1
```

Comment:
I know `components` returns a string inside an array even if the original string doesn't contain the argument passed. In this case, is it okay to use force-unwrapping since I know that `splitRange` will definitely contain both a `.first` and a `.last` element?